e2e/app/element/base-element.ts

```diff
@@ -142,8 +142,28 @@ export default class BaseElement extends Container {
 
   async click(options?: ClickOptions): Promise<void> {
     return this.asElementHandle()
-      .then(elemt => {
-        return elemt.click(options);
+      .then(async element => {
```

Comment:
Experiment: 
Sometimes click "Confirm" button on "Create Workspace" modal window seems to be broken. A recent failure [here](https://app.circleci.com/pipelines/github/all-of-us/workbench/7125/workflows/4cd3d050-c798-4b37-b44c-1b2c888a6b01/jobs/118074/parallel-runs/1?filterBy=FAILED). 
If similar failure won't be happening anymore, I'll keep this change. Otherwise, I'll remove this wait & check and look for another alternative.

Reply:
Good call. I'll make a separate PR for the new function. I'll remove this change in this PR.